docs/docs-content/release-notes/release-notes.md

```diff
@@ -99,7 +106,7 @@ to learn more about this release's new features and improvements.
 #### Features
 
```

Comment:
- To lines 83-84: Maybe would be a good idea to make this an admonition? Seems like a bit of information that should stand out :)
- To line 92: InPlaceUpgrade is missing code formatting 👀